calendar.js

```diff
@@ -1,11 +1,75 @@
 //function to display the table
-function myTable() {
-    var table = createTable();
-    getScheduleAndUpdateTable(table);
+function displayCalendar() {
+
+    createPrevAndNextLinks();
+    if(document.getElementById('calendar-id') != null && document.getElementById('calendar-id').dataset.sundayepochtime != undefined){
+        var table = createCalendarView('calendar-id', document.getElementById('calendar-id').dataset.sundayepochtime);
+    }
+    else{
+     var table = createCalendarView('calendar-id', getSundayOfCurrentWeekInEpoch(new Date()));   
+    }
+    getScheduleAndUpdateTable(table, getSundayOfCurrentWeekInEpoch(new Date()), getSaturdayOfCurrentWeekInEpoch(new Date()));
     document.body.appendChild(table);
+}
+
+displayCalendar();
```

Comment:
move this function call to the bottom of this file (like how we write the `main` function in the end in C and C++ code)